tests/end2end/features/misc.feature

```diff
@@ -671,3 +671,38 @@ Feature: Various utility commands.
         And I wait until the scroll position changed
         And I run :click-element id link
         Then the error "Element position is out of view!" should be shown
+
+    ## :command-history-{prev,next}
+    
+    Scenario: Calling previous command
+        When I run :set-cmd-text :message-info blah
+        And I run :command-accept
+        And I wait for "blah" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-accept
+        Then the message "blah" should be shown
+    
+    Scenario: Browsing through commands 
+        When I run :set-cmd-text :message-info blarg
+        And I run :command-accept
+        And I wait for "blarg" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-history-prev
```

Comment:
I'm not sure what you mean by 'isolation'. All tests pass when run with 
`tox -e py35 -- tests/end2end/features/test_misc_bdd.py -k command`. But let's wait for Travis or AppVeyor to finish.

However, I do have a problem with last two tests. They seem to pass, but throw [errors](https://gist.github.com/sebastian-frysztak/5ed41740ac66e35ca650e8010839c903). I think it might be due to lack of NetworkManager on my system (I use ConnMan).